internal/app/model/entities.go

```diff
@@ -1,4 +1 @@
 package model
```

Comment:
If there is no code then should we keep the file?

Reply:
Also, there are some other files and folders which are empty, if are not needed yet, we should remove them.